catalog/nsili/catalog-nsili-orb-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,7 +17,7 @@
     <OCD name="NSILI Corba ORB" id="org.codice.alliance.nsili.orb.impl.corbaorb"
          description="NSILI Corba ORB Configuration">
 
-        <AD description="Corba Listen Port."
+        <AD description="Corba Listen Port. NOTE: A restart is required to set the port back to a port number that has already been set."
```

Comment:
After talking with @troymohl, it looks like the orb doesn't quite ever give up the post until a restart and there doesn't seem to be a satisfactory solution for this.